src/main/java/com/mythsoftware/rest/resource/UserResource.java

```diff
@@ -0,0 +1,40 @@
+package com.mythsoftware.rest.resource;
+
+import com.mythsoftware.rest.model.User;
+import com.mythsoftware.rest.service.UserService;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+/**
+ */
+@Named
+@Path("/user")
+public class UserResource {
+
+    @Inject
+    private UserService userService;
+
+    @GET
+    @Path("/getUsers")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getUsers() {
+        List<User> users = userService.getUsers();
+        return Response.ok(users).build();
+    }
+
+    @GET
+    @Path("/getUserById/{id}")
```

Comment:
OK, I'll change that when I add some more stuff. What are most people used
to when they post/put/patch/delete?

On Thu, Jun 11, 2015 at 5:17 PM, jasonmullins <notifications@github.com>
wrote:

> In src/main/java/com/mythsoftware/rest/resource/UserResource.java
> <https://github.com/MythSoftware/aurora/pull/5#discussion_r32269429>:
>
> > +@Path("/user")
> > +public class UserResource {
> > +
> > +    @Inject
> > +    private UserService userService;
> > +
> > +    @GET
> > +    @Path("/getUsers")
> > +    @Produces(MediaType.APPLICATION_JSON)
> > +    public Response getUsers() {
> > +        List<User> users = userService.getUsers();
> > +        return Response.ok(users).build();
> > +    }
> > +
> > +    @GET
> > +    @Path("/getUserById/{id}")
>
> And here the path would be something like this:
> GET /users/{id}
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MythSoftware/aurora/pull/5/files#r32269429>.
>